test/views/EditSetRecordEditDiscardViewSpec.scala

```diff
@@ -21,33 +21,35 @@
 
 package views
 
-import org.jsoup.nodes.Document
 import play.api.i18n.Messages
 import play.api.test.Helpers
-import play.twirl.api.Html
 import support.BaseSpec
 import support.CommonMatchers._
+import uk.gov.hmrc.govukfrontend.views.html.components.GovukNotificationBanner
 import uk.gov.nationalarchives.omega.editorial.models.PhysicalRecord
 import uk.gov.nationalarchives.omega.editorial.views.html.editSetRecordEditDiscard
 
-class EditSetRecordEditDiscardSpec extends BaseSpec {
+class EditSetRecordEditDiscardViewSpec extends BaseSpec {
 
```

Comment:
Should this extend `BaseViewSpec` for consistency?

Reply:
Good idea.

Reply:
I've made this change now.